### ISSUE TITLE
Example app should default to debug=True

### ISSUE THREAD

**Issue by pauleveritt** [florimondmanca/arel#18]
The example doesn't work out-of-the-box, as changes aren't reloaded. IMO, this line in `examples/server/settings.py` should default to `True`:

```python
DEBUG = config("DEBUG", cast=bool, default=False)
```

As background, I was working very slowly on writing just this very thing: a reloader more sophisticated/performant than `livereload`, based on the `watchgod` changeset approach, in Starlette. But learning asyncio was really slowing me down. Thanks!

**Comment by florimondmanca** [comment]
@pauleveritt Hello! This suggestion sounds appropriate to me. Happy to review a PR if you're down for it.

**Comment by pauleveritt** [comment]
Yep, I'm down for it. Mind if I (in another PR) make a second example which takes the changeset and only changes things in the tree that were add/edit/delete?

**Comment by florimondmanca** [comment]
Not sure what that means exactly, but as a separate PR, sure. :)

**Comment by y2kbugger** [comment]
> Yep, I'm down for it. Mind if I (in another PR) make a second example which takes the changeset and only changes things in the tree that were add/edit/delete?

@pauleveritt see #37
